## GitHub Page of Exposome and Heredity Team at CESP

This is the GitHub page for the Exposome and Heredity Team at Centre for Epidemiology and Population Health (CESP), France.
<br>
<br>
https://cesp.inserm.fr/en/equipe/exposome-and-heredity
<br>
<br>

Here are some of our most popular repositories and a brief description:

| Title                                                                                                                                          | Description                                                            |
|:-----------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------------------------------------------------|
| [GCPBays Pipeline](https://github.com/CESP-ExpHer/GCPBayes-Pipeline) | GCPBayes Pipeline is a tool for exploring pleiotropy at gene-level |
| **Some Tutorials** |   |
| [Annotate GWAS Data](https://github.com/CESP-ExpHer/Gene_Annotation) | Annotation of GWAS data using an annotation file: A comprehensive survey considering annotation files from different databases |
| [ANNOVAR](https://github.com/CESP-ExpHer/Tutorial_ANNOVAR) | A tool for Functional annotation of genetic variants |
| [ASSET](https://github.com/CESP-ExpHer/Tutorial_ASSET) | ASSET for exploring Pleiotropy |
| [Docker](https://github.com/CESP-ExpHer/Tutorial_Docker) **(Private)** | Tutorial for working with Docker |
| [EPIC](https://github.com/CESP-ExpHer/Association-test-on-EPIC) **(Private)** | Association tests results on EPIC data |
| [Epithyr Info](https://github.com/CESP-ExpHer/Association-test-on-Epithyr) **(Private)** | Association tests results on Epithyr data (different versions) |
| [Epithyr-EPIC](https://github.com/CESP-ExpHer/Meta-analysis-Epithyr-EPIC) **(Private)** | Meta-analysis result of Epithyr and EPIC data |
| [Extracting SNP info](https://github.com/CESP-ExpHer/dbSNP_crawling) | A python code to retrieve the chromosome & position of an SNP (rsID) from NCBI or vice versa |
| [GPC](https://github.com/CESP-ExpHer/GPC) | Tutorial for working with GPC (an R package for calculating the GWAS power based on the odds ratio, minor allele frequency, and number of samples) |
| [GWAS Standardization](https://github.com/CESP-ExpHer/GWAS-Standardization) | Standardization of GWAS Summary Statistics data for two traits |
| [GWAS Standardization 2](https://github.com/CESP-ExpHer/Harmonize) | A tutorial for performing harmonization of GWAS summary statistics |
| [Liftover GWAS](https://github.com/CESP-ExpHer/Tutorial_GWAS_Human_Assembly_Liftover) | GWAS Data Liftover (from hg19 to hg38 and vice versa) |
| [Liftover Genes](https://github.com/CESP-ExpHer/Tutorial_Genes_Human_Assembly_Liftover) | Genes Data Liftover (from hg19 to hg38 and vice versa) |
| [LocusZoom](https://github.com/CESP-ExpHer/locuszoom) **(Private)** | A tutorial explaining how to use LocusZoom tool available in our local server for your GWAS data |
| [mrMediator](https://github.com/CESP-ExpHer/mrMediator) | A tutorial to run Mediator MR  |
| [MVMR](https://github.com/CESP-ExpHer/MVMR) **(Private)** | A tutorial to run Multivariable Mendelian randomization analysis |
| [MR-TwoSample](https://github.com/CESP-ExpHer/TwoSample-MR) **(Private)** | A tutorial to provide two sample Mendelian Randomization |
| [Meta-Analysis with GWAMA](https://github.com/CESP-ExpHer/Meta-Analysis_with_GWAMA) **(Private)** | A tutorial to run GWAS Meta-Analysis with GWAMA |
| [PLACO](https://github.com/CESP-ExpHer/Tutorial_PLACO) | PLACO for exploring Pleiotropy |
| [Plink](https://github.com/CESP-ExpHer/GWAS_with_Plink) **(Private)** | A tutorial to run logistic regression using PLINK on GWAS individual-level data |
| [Programming Tips](https://github.com/CESP-ExpHer/Programming_Tips) | Some useful tips for programming (ex., a Fast Way to read or write Big Text Data in R, dplyr Library for Fast Operations in R, Calculation of Very Small Values for P-value based on beta and se) |
| [Proxy SNP](https://github.com/CESP-ExpHer/dbSNP_crawling) | A python code to retrieve the proxy SNP of your SNP of interest from LDlink |
| [UKBB Procedure](https://github.com/CESP-ExpHer/UKBB-PROCEDURES) **(Private)** | Some tutorials describing procedures for working with the UKBB |
| [Working with Server](https://github.com/CESP-ExpHer/Working_with_Server) **(Private)** | some information for the ones who want to work on the CESP-ExpHer Linux Server |

